.changeset/dialog-drawer-event-propagation.md

```diff
@@ -0,0 +1,5 @@
+---
+'svelte-ux': patch
+---
+
+Fixed (Dialog/Drawer) event propagation preventing outside click detection
```

Comment:
Can you update the changeset to: 
```
fix(Dialog/Drawer): event propagation preventing outside click detection
```

I try to follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) formatting for changesets, similar to [Svelte](https://github.com/sveltejs/svelte/releases) / others